### PR TITLE
db/default-menu-icon

### PR DESCRIPTION
**Pull request by karolkolodziej**
Added default icon, and fixed CSP.

### CODE REVIEW

**Comment by bre1470** [comment]
This is working neither.

Why did you removed the internal graphics? Shouldn't we go with own SVGs instead of creating a boostrap dependency?

**Comment by karolkolodziej** [comment]
The SVG from `https://code.highcharts.com/@product.version@/gfx/dashboard-icons/menu.svg ` is our icon located in the `dashboard-icon` folder.
But since this is not live yet there is nothing under this URL.

When it comes to `bootstrap` it looks like in the demo we are trying to create a similar dashboard with it.
Perhaps this should go into a separate folder/demo- what do you think @sebastianbochan ?

File: samples/highcharts/studies/dashboard-layout/demo.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@import "https://code.highcharts.com/css/dashboard/gui.css";
```
Comment (bre1470) [inline samples/highcharts/studies/dashboard-layout/demo.css:1]: `studies/dashboard-layout` should not exist anymore. Please merge it with `dashboard/demos/dashboard-layout`.

File: samples/dashboard/demos/dashboard-layout/demo.html [hunk 2]
```diff
@@ -1,7 +1,5 @@
-<link rel="stylesheet" type="text/css" href="https://code.highcharts.com/css/dashboard/gui.css">
-
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
```
Comment (bre1470) [inline samples/dashboard/demos/dashboard-layout/demo.html:1]: If this is a bootstrap demo the folder name should change. Dashboard itself should work without third-party libraries.

**Review by bre1470 — CHANGES_REQUESTED** [review]
The icon change looks good. 👍  Other changes need a discussion.

File: samples/dashboard/demos/dashboard-layout/demo.html (hunk 2 above)
Comment (karolkolodziej) [reply samples/dashboard/demos/dashboard-layout/demo.html:1]: Created ticket in Asana

**Review by pawellysy — APPROVED** [review]
lgtm :)

**Pull request merged**